src/creates/actor_run.js

```diff
@@ -0,0 +1,70 @@
+const { APIFY_API_ENDPOINTS, ACTOR_RUN_SAMPLE, ACTOR_RUN_OUTPUT_FIELDS } = require('../consts');
+const { enrichActorRun, getActorAdditionalFields } = require('../apify_helpers');
+const { wrapRequestWithRetries } = require('../request_helpers');
+
+const runActor = async (z, bundle) => {
+    const { actorId, runSync, inputBody, inputContentType, build, timeoutSecs, memoryMbytes } = bundle.inputData;
+
+    const requestOpts = {
+        url: `${APIFY_API_ENDPOINTS.actors}/${actorId}/runs`,
+        method: 'POST',
+        params: {
+            build,
+            timeout: timeoutSecs,
+            memory: memoryMbytes,
+        },
+    };
+
+    if (runSync) requestOpts.params.waitForFinish = 120;
+    if (inputContentType) {
+        requestOpts.headers = {
+            'Content-Type': inputContentType,
+        };
+    }
+    if (inputBody) requestOpts.body = inputBody;
+
+    const runResponse = await wrapRequestWithRetries(z.request, requestOpts);
+
+    let run = runResponse.json;
+    if (runSync) {
+        run = await enrichActorRun(z, run);
```

Comment:
You can't be sure that run finished at this point. Should we throw some error? Or wait another 120s? what is the limit?

Reply:
I have to test it in Zapier which error we can get on UI. I will update this in next version.